definitions/procedures/packages/check_for_reboot.rb

```diff
@@ -0,0 +1,18 @@
+module Procedures::Packages
+  class CheckForReboot < ForemanMaintain::Procedure
+    metadata do
+      description 'Check if system needs reboot'
+    end
+
+    def run
+      status, output = execute_with_status('dnf needs-restarting --reboothint')
+      puts output
+    rescue ForemanMaintain::Error::ExecutionError => e
+      if @warn_on_errors
```

Comment:
I don't see `@warn_on_errors` being defined anywhere here. so it will always be `nil`